tool/build_xcframework.sh

```diff
@@ -3,12 +3,11 @@ set -e
 
 # Adapted from https://github.com/vlcn-io/cr-sqlite/blob/main/core/all-ios-loadable.sh
 
-
 BUILD_DIR=./build
-DIST_PACKAGE_DIR=./dist
 
 function createXcframework() {
-  plist=$(cat << EOF
+  ios_plist=$(
```

Comment:
It's technically not just ios, we're using this for macOS too. I don't know what a better name would be either though, maybe `common_plist`?